src/js/languages/fa.js

```diff
@@ -0,0 +1,90 @@
+/*!
+ * This is a `i18n` language object.
+ *
+ * Persian
+ *
+ * @author
+ *   wmateam (GitHub: @wmateam)
+ *
+ * @see core/i18n.js
+ */
+
+(function (exports) {
+    if (exports.fa === undefined) {
+        exports.fa = {
+            "mejs.plural-form": 1,
```

Comment:
I`m sorry,my friend.The problem was solved.
and soon i check https://github.com/mediaelement/mediaelement-plugins and test it.